Pod/Support/iOS/simctl

```diff
@@ -0,0 +1,72 @@
+#!/bin/sh
+
+APPLICATION_PATH=$1
+SLIMSERVER_PORT=$2
+APP_IDENTIFIER=$(defaults read $APPLICATION_PATH/Info CFBundleIdentifier)
+
+check_device_booted() {
+	xcrun simctl list devices | grep -i booted
+}
+
+boot_device() {
+	DEVICE_ID=$(get_device_id)
+	log "Found Device $(xcrun simctl list devices | grep $DEVICE_ID)"
+	log "Booting $DEVICE_ID"
+	xcrun simctl boot $DEVICE_ID
+}
+
+get_device_id() {
+    xcrun simctl list devices | grep -i iphone | grep -i -v unavailable | tail -n1 | cut -d "(" -f2 | cut -d ")" -f1
+}
+
+log() {
+	echo "[OCSP_SIMCTL] $1"
+}
+
+preflight() {
+	if ! check_device_booted; then 
+		log "No Device Booted"
+		boot_device
+	fi
+}
+
+install() {
+	log "Installing $APP_IDENTIFIER"
+	xcrun simctl install booted $APPLICATION_PATH
+}
+
+
+launch() {
+	log "Launching... (Slim Port: $SLIMSERVER_PORT)"
+	xcrun simctl launch booted $APP_IDENTIFIER $SLIMSERVER_PORT
+}
+
+main() {
+	log "== Launching Slim Host App $(xcrun simctl --version)  =="
+	preflight
+	install
+	launch
+	exit $?
+}
+
+main
+
+
+
+# Command line utility to control the Simulator
```

Comment:
`$simctl [--help | -h]`?

Then, you wouldn't need to find and open this file to see the usage.